test/yaml_edit/fuzz.dart

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:pub/src/yaml_edit/yaml_edit.dart';
+import 'package:pub/src/yaml_edit/src/wrap.dart';
+import 'package:yaml/yaml.dart';
+
+import 'problem_strings.dart';
+import 'test_utils.dart';
+
+/// Performs naive fuzzing on an initial YAML file based on an initial seed.
+void main(List<String> args) {
```

Comment:
Can you write a few lines explaining what it does, and maybe suggest why we don't run it as part of normal integration testing...

Could we run it with just a few rounds in a `_test.dart` file?

Reply:
modified this to `fuzz_test.dart` to run 10 rounds of 50 modifications each!